docs/.prettierrc.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
   endOfLine: "auto",
+  trailingComma: "all",
```

Comment:
for the sake of this PR I would suggest:
```suggestion
  trailingComma: "es5",
```
that'll reduce the noise from all the extra commas. (better to keep linting and formatting as separate PRs)